Plugins/BuildServerIntegration/AzureDevOpsIntegration/Settings/ProjectUrlHelper.cs

```diff
@@ -101,7 +102,7 @@ private static (bool success, string result) TryTransformString(string value, Di
         /// <returns>
         /// A tuple that contains whether a Azure DevOps / TFS project could be recognized from the given url and the resulting home page url of the project.
         /// </returns>
-        public static (bool success, string projectUrl) TryDetectProjectFromRemoteUrl(string remoteUrl)
+        public static (bool success, string projectUrl) TryDetectProjectFromRemoteUrl([CanBeNull] string remoteUrl)
```

Comment:
Added instances of the JetBrains annotations here (and elsewhere) for projects which are not yet annotated, to make annotating them easier in future.